Update windows build docs

kylewu opened the pull request:
Fixes #3764

tpwwu committed:
vs 2017 in install doc

kylewu commented:
There are another two files could be updated, should I push them in another commit?

```
index 5fd8548..729c029 100644
--- a/appveyor.yml
+++ b/appveyor.yml
@@ -31,7 +31,7 @@ branches:
   only:
     - release
     - develop
-os: Visual Studio 2015
+os: Visual Studio 2017
 configuration:
     - RelWithDebInfo
 environment:
@@ -62,7 +62,7 @@ install:
 before_build:
     - if not exist build mkdir build
     - cd build
-    - cmake -G "Visual Studio 14 2015 Win64" .. -DTESTS=On
+    - cmake -G "Visual Studio 15 2017 Win64" .. -DTESTS=On
 build_script:
     - msbuild solidity.sln /p:Configuration=%CONFIGURATION% /m:%NUMBER_OF_PROCESSORS% /v:minimal
     - cd %APPVEYOR_BUILD_FOLDER%
diff --git a/scripts/release.bat b/scripts/release.bat
index 54d8586..1060eda 100644
--- a/scripts/release.bat
+++ b/scripts/release.bat
@@ -30,4 +30,4 @@ set CONFIGURATION=%1
 
 7z a solidity-windows.zip ^
     .\build\solc\%CONFIGURATION%\solc.exe .\build\test\%CONFIGURATION%\soltest.exe ^
-    "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\redist\x86\Microsoft.VC140.CRT\msvc*.dll"
+    "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Redist\MSVC\14.13.26020\x86\Microsoft.VC141.CRT\msvc*.dll"


```

chriseth commented:
Yes, please do that in a different commit, but please (if that is possible with appveyor), make it run both versions instead of just one version.

chriseth commented on docs/installing-solidity.rst:
```diff
@@ -210,12 +210,30 @@ You will need to install the following dependencies for Windows builds of Solidi
++------------------------------+-------------------------------------------------------+
+
+Visual Studio 2017 Build Tools provides all required compiler and libraries.
+If you prefer your own IDE, Visual Studio 2017 Build Tools is a perfect choice.
```
I'm not sure if we are at liberty to say that it is a "perfect choice".

Could you please tone down this and the next paragraph so that we just suggest this as an IDE, but do not praise it? :)

chriseth commented on docs/installing-solidity.rst:
```diff
@@ -210,12 +210,30 @@ You will need to install the following dependencies for Windows builds of Solidi
 +------------------------------+-------------------------------------------------------+
 | `CMake`_                     | Cross-platform build file generator.                  |
```
The `|` has to be moved to the right now, I think. Otherwise, the table does not render correctly.

kylewu commented:
@chriseth i updated the text and table.

Regarding appveyor, i updated yml based on https://www.appveyor.com/docs/build-environment/#using-multiple-images-for-the-same-build how can i test my change?


update:
ah, i see jobs here https://ci.appveyor.com/project/ethereum-free/solidity/build/1.0.8007

kylewu commented:
@chriseth could release.bat be used locally? Or only in appveyor?

chriseth commented:
I'm sorry, what I meant was: Can you please split the documentation change and the bulid system change into two different pull requests (not only commits)?

kylewu commented:
@chriseth no problem. here is the pr for appveyor change https://github.com/ethereum/solidity/pull/4052
